tests/Executor/ResolveTest.php

```diff
@@ -33,7 +33,7 @@ public function testDefaultFunctionAccessesProperties(): void
     }
 
     /**
-     * @param UnnamedFieldDefinitionConfig $testField
+     * @phpstan-param UnnamedFieldDefinitionConfig $testField
```

Comment:
Why?

Reply:
No idea, it was failing for no reason. + For consistency, everywhere else its used like `phpstan-param` not `param.